packages/@tailwindcss-upgrade/src/template/codemods/important.test.ts

```diff
@@ -15,5 +15,25 @@ test.each([
     base: __dirname,
   })
 
-  expect(important(designSystem, {}, candidate)).toEqual(result)
+  expect(
+    important(designSystem, {}, candidate, {
+      contents: `"${candidate}"`,
+      start: 1,
+      end: candidate.length + 1,
+    }),
+  ).toEqual(result)
+})
+
+test('does not match false positives', async () => {
+  let designSystem = await __unstable__loadDesignSystem('@import "tailwindcss";', {
+    base: __dirname,
+  })
+
+  expect(
+    important(designSystem, {}, '!border', {
+      contents: `let notBorder = !border\n`,
```

Comment:
If it's super easy, could we make this work with trailing whitespace too? Like `!border    \n`, for people not using Prettier or who don't have their editor configured to trim trailing whitespace and have stuff like this in there by mistake.

If it's too hard no big deal though.